libpod/sql_state.go

```diff
@@ -105,9 +105,6 @@ func (s *SQLState) Refresh() (err error) {
                              Mountpoint=?,
                              Pid=?;`
 
-	s.lock.Lock()
-	defer s.lock.Unlock()
-
 	if !s.valid {
```

Comment:
Maybe I missed this earlier, but just wanted to check that we're confident in removing this lock.  Will the db provide sufficient locking itself?

Reply:
yes, the db provides sufficient locking on its own, so the Lock() method was removed.

Reply:
This one is actually breaking the build in master, so it's not really related to this code except for the fact that it's breaking the build everywhere. Oops?

Reply:
yeah, I guess it was lingering after the Lock() method was removed. It will be fixed once this gets in.

Reply:
👍 Thanks for the locking verification.